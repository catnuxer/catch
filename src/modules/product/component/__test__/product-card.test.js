import React from "react";
import serializer from "jest-emotion";
import renderer from "react-test-renderer";
import Card from "../product-card";

expect.addSnapshotSerializer(serializer);

describe("product-card test", () => {
  it("snappshoot", () => {
    const ProductCard = () => {
      return <Card.Col></Card.Col>;
    };
    expect(renderer.create(<ProductCard />).toJSON()).toMatchSnapshot();
  });
});
