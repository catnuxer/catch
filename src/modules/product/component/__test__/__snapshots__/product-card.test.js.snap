// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout-container test snappshoot 1`] = `
.emotion-0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  max-width: 25%;
  -webkit-flex: 0 1 25%;
  -ms-flex: 0 1 25%;
  flex: 0 1 25%;
}

@media only screen and (min-width :1824px) {
  .emotion-0 {
    max-width: 20%;
    -webkit-flex: 0 1 20%;
    -ms-flex: 0 1 20%;
    flex: 0 1 20%;
  }
}

@media only screen and (min-width :768px) and (max-width :1024px) {
  .emotion-0 {
    max-width: 33%;
    -webkit-flex: 0 1 33%;
    -ms-flex: 0 1 33%;
    flex: 0 1 33%;
  }
}

@media only screen and (min-width :320px) and (max-width :768px) {
  .emotion-0 {
    max-width: 50%;
    -webkit-flex: 0 1 50%;
    -ms-flex: 0 1 50%;
    flex: 0 1 50%;
  }
}

<div
  className="emotion-0"
/>
`;

exports[`product-card test snappshoot 1`] = `
.emotion-0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  max-width: 25%;
  -webkit-flex: 0 1 25%;
  -ms-flex: 0 1 25%;
  flex: 0 1 25%;
}

@media only screen and (min-width :1824px) {
  .emotion-0 {
    max-width: 20%;
    -webkit-flex: 0 1 20%;
    -ms-flex: 0 1 20%;
    flex: 0 1 20%;
  }
}

@media only screen and (min-width :768px) and (max-width :1024px) {
  .emotion-0 {
    max-width: 33%;
    -webkit-flex: 0 1 33%;
    -ms-flex: 0 1 33%;
    flex: 0 1 33%;
  }
}

@media only screen and (min-width :320px) and (max-width :768px) {
  .emotion-0 {
    max-width: 50%;
    -webkit-flex: 0 1 50%;
    -ms-flex: 0 1 50%;
    flex: 0 1 50%;
  }
}

<div
  className="emotion-0"
/>
`;
